build/implementations/get-implementation-for-report.js

```diff
@@ -0,0 +1,27 @@
+const getAssertions = require('./get-assertions')
+const getTestcasesGroupedByRule = require('./get-testcases-grouped-by-rule')
+const getRuleMapping = require('./get-rule-mapping')
+
+/**
+ * Get implementation metric from submitted report
+ *
+ * @param {Object|Array<Objects>} reports implementation reports
+ */
+const getImplementationForReport = async reports => {
+	const assertions = getAssertions(reports)
+	const testcasesGroupedByRuleId = await getTestcasesGroupedByRule()
+
+	return Object.keys(testcasesGroupedByRuleId)
+		.map(ruleId => {
+			const ruleTestcases = testcasesGroupedByRuleId[ruleId]
```

Comment:
If you use `Object.entries()`, you get a tuple of `[key, value]`. You don't have to do the extra lookup.